Dockerfile.trigger_archivematica

```diff
@@ -0,0 +1,40 @@
+FROM public.ecr.aws/lambda/nodejs:18 as builder
+WORKDIR /usr/local/apps/stela/
+
+COPY package.json ./
+COPY tsconfig.build.json ./
+COPY tsconfig.json ./
+COPY jest.config.js ./
+COPY packages ./packages
+
+RUN npm install -g npm@8.19.3
```

Comment:
I think we missed this one (unless it *is* needed in which case please disregard!)